pallets/exchange/Cargo.toml

```diff
@@ -0,0 +1,63 @@
+[package]
+name = "pallet-exchange"
```

Comment:
confused that this is showing up here :thinking:

Reply:
math and warehouse seem fine, no? would prefer not to lose the history there
I can do this here, yeah